docs/content/developers/building-contributing.md

```diff
@@ -114,82 +116,83 @@ To manually push using GitHub Actions,
+Adapted from [GitHub Guides](https://guides.github.com/activities/contributing-to-open-source/).
 
 ## Coding Style
 
 Unless explicitly stated, we follow all coding guidelines from the Go community. While some of these standards may seem arbitrary, they somehow seem to result in a solid, consistent codebase.
 
 It is possible that the code base does not currently comply with these guidelines. We are not looking for a massive PR that fixes this since that goes against the spirit of the guidelines. All new contributions should make a best effort to clean up and make the code base better than they left it. Obviously, apply your best judgment. Remember, the goal here is to make the code base easier for humans to navigate and understand. Always keep that in mind when nudging others to comply.
 
-Just use `make staticrequired` to ensure that your code can pass the required static analysis tests.
+Use `make staticrequired` to ensure that your code can pass the required static analysis tests.
 
 The rules:
 
 1. All code should be formatted with `gofmt -s`.
 2. All code should pass the default levels of [`golint`](https://github.com/golang/lint).
 3. All code should follow the guidelines covered in [Effective Go](http://golang.org/doc/effective_go.html) and [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments).
 4. Comment the code. Tell us the why, the history and the context.
-5. Document _all_ declarations and methods, even private ones. Declare expectations, caveats and anything else that may be important. If a type gets exported, having the comments already there will ensure it's ready.
+5. Document *all* declarations and methods, even private ones. Declare expectations, caveats and anything else that may be important. If a type gets exported, having the comments already there will ensure it’s ready.
 6. Variable name length should be proportional to its context and no longer. `noCommaALongVariableNameLikeThisIsNotMoreClearWhenASimpleCommentWouldDo`. In practice, short methods will have short variable names and globals will have longer names.
 7. No underscores in package names. If you need a compound name, step back, and re-examine why you need a compound name. If you still think you need a compound name, lose the underscore.
-8. All tests should run with `go test` and outside tooling should not be required. No, we don't need another unit testing framework. Assertion packages are acceptable if they provide _real_ incremental value.
-9. Even though we call these "rules" above, they are actually just guidelines. Since you've read all the rules, you now know that.
+8. All tests should run with `go test` and outside tooling should not be required. No, we don’t need another unit testing framework. Assertion packages are acceptable if they provide *real* incremental value.
+9. Even though we call these “rules” above, they are actually just guidelines. Since you’ve read all the rules, you now know that.
```

Comment:
```suggestion
9. Even though we call these "rules" above, they are actually just guidelines. Since you’ve read all the rules, you now know that.
```

Again, please use straight quotes and apostrophes everywhere. All editors know what to do with them.